README.md

```diff
@@ -58,6 +58,19 @@ const registry = new Registry();
 
 // create a circuit
 const circuit = new CircuitBreaker(functionThatMightFail);
-const metrics = new PrometheusMetrics(circuit, registry)
+const metrics = new PrometheusMetrics({ circuits: [circuit], registry: registry })
 ```
 
+## Options
+The second argument of the `PrometheusMetrics` constructor is an options object.
+
+```js
+const options = {};
+new PrometheusMetrics(options)
+```
+
+|Name                      |Description                                                             |Default       |
+|--------------------------|------------------------------------------------------------------------|--------------|
+|`circuits`                |A list or individual circuit breaker to create metrics for              |No circuits   |
+|`registry`                |An existing registry to use for prometheus metrics                      |A new registry|
```

Comment:
i think the default here is undefined, or something like that

Reply:
I have added some more information about this option